app/models/movie.rb

```diff
@@ -1,3 +1,13 @@
 class Movie < ApplicationRecord
   belongs_to :studio
+  has_many :movie_actors
+  has_many :actors, through: :movie_actors
+
+  def actors_ordered_by_name_age
+    actors.order(name: :asc, age: :asc)
```

Comment:
Great job making these AR methods! However, we do want to have model tests for each new method you create, as well as any associations/relationships.